### PR TITLE
feat/hw-10

### PR DESCRIPTION
**Pull request by YaninaAQA**
hw-10

### CODE REVIEW

File: tests/loan.spec.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    expect.soft(responseBody.riskLevel).toBe('Low Risk')
+  })
+
+  test('POST loan: returns 200 status code with medium risk', async ({ request }) => {
+    const riskDto = LoanDto.calcPositiveRiskScore()
+    riskDto.loanPeriod = 9
+    const response = await request.post(
+      'https://backend.tallinn-learning.ee/api/loan-calc/decision',
+      {
+        data: riskDto,
```
Comment (niamh-d) [inline tests/loan.spec.ts]: Just a tip. You could use the spread syntax https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

and rewrite line 26 as data: {...riskDto, loanPeriod: 9}
and then delete line 22 and you've saved yourself a line :-)

File: tests/loan.spec.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      },
+    )
+    const responseBody = await response.json()
+    expect.soft(response.status()).toBe(StatusCodes.OK)
+    expect.soft(responseBody.riskLevel).toBe('High Risk')
+  })
+  test('POST loan: returns 200 status code with negative decision', async ({ request }) => {
+    const riskDto = LoanDto.calcPositiveRiskScore()
```
Comment (niamh-d) [inline tests/loan.spec.ts]: This is confusing for me. The dto is for a positive risk score but you're using it in a negative decision test. Your dto already has a negative risk score method. I would suggest changing this.

Comment (vladimir-tl) [reply tests/loan.spec.ts]: +1

File: tests/loan.spec.ts (hunk 1 above)
Comment (YaninaAQA) [reply tests/loan.spec.ts]: Fixed

File: tests/loan.spec.ts (hunk 2 above)
Comment (YaninaAQA) [reply tests/loan.spec.ts]: Fixed

**Pull request merged**